lotify/client.py

```diff
@@ -170,3 +171,42 @@ def __check_error(response):
             pass
         else:
             raise ValueError(response.json())
+
+def main(args):
+    client = Client()
+    if args.t:
+        access_token=args.t
+    if args.m and (not args.imgF):
+        message=args.m
+        response=client.send_message(access_token,message)
+        print(response)
+    if args.imgF and args.m:
+        message=args.m
+        image_filename=args.imgF
+        image = client.send_message_with_image_file(
+        access_token=access_token,
+        message=message,
+        file=open('./'+image_filename+'.jpg', 'rb')   #TODO : support absolute PATH
+)
+        print(image)
+    
+    
+
+if __name__ == '__main__' :
+    parser=argparse.ArgumentParser()
+    parser.add_argument("-t", help="input yout token",type=str)  #nargs allows multiple args
+    parser.add_argument("-m", help="send text message",type=str)
+    parser.add_argument("-imgF", help="send message by File",type=str) 
+
+    '''
```

Comment:
Hi @dfgh012316 , would you like to take your time implement it?

Reply:
Sure , I will try to implement it .
I want to contribute my code to this project .

Reply:
ok, thanks for take your times